### PR TITLE
Paramaterize Prometheus integration test to use for local testing

### PR DESCRIPTION
**Pull request by robskillington**

### CODE REVIEW

File: scripts/integration-tests/prometheus/prometheus-integration-test.sh [hunk 1]
```diff
@@ -5,9 +5,16 @@ set -xe
 rm -rf /tmp/m3dbdata/
 mkdir -p /tmp/m3dbdata/
 
-echo "Build docker images"
+PARAM_TEST_BUILD="${TEST_BUILD:-true}"
```
Comment (nerd0) [inline scripts/integration-tests/prometheus/prometheus-integration-test.sh]: Are these injected by semaphore? Or just more meant for local testing?

File: scripts/integration-tests/prometheus/docker-compose.yml [hunk 2]
```diff
@@ -38,5 +38,13 @@ services:
     image: prom/prometheus:latest
     volumes:
       - "./:/etc/prometheus/"
+  grafana:
```
Comment (nerd0) [inline scripts/integration-tests/prometheus/docker-compose.yml]: Just curious if grafana used within the test?

Comment (robskillington) [reply scripts/integration-tests/prometheus/docker-compose.yml]: It is not, we're going to be using this as a hack to run this locally heh.

File: scripts/integration-tests/prometheus/prometheus-integration-test.sh (hunk 1 above)
Comment (robskillington) [reply scripts/integration-tests/prometheus/prometheus-integration-test.sh]: Local testing yeah.

**Pull request merged**